tests/test_dataset.py

```diff
@@ -288,15 +334,20 @@ def test_graph_save_hist(self):
         assert len(os.listdir(output_directory)) > 0
 
         rmtree(output_directory)
-
-    def test_graph_standardize(self):
+    
+    def test_standardize_graphdataset(self):# noqa: MC0001, pylint: disable=too-many-locals
 
         hdf5_path = "tests/data/hdf5/train.hdf5"
 
+        features_transform = {'bsa': {'standardize': True},
+                        'sasa': {'standardize': True},
+                        'hb_donors':{'standardize': False},
+                        'hse': {'standardize': True}}
+        
         dataset = GraphDataset(
             hdf5_path = "tests/data/hdf5/train.hdf5",
-            target='binary',
-            standardize=True
+            target = 'binary',
+            features_transform = features_transform
         )
 
         with h5py.File(hdf5_path, 'r') as f5:
```

Comment:
This with statement until the end of `features_dict` population can be wrapped up in an internal function called like `get_features(hdf5_path, dataset)`; it will be more readable this way and we'll use this function for testing the transformations as well.